core/store/src/trie/trie_storage.rs

```diff
@@ -693,10 +694,13 @@ mod trie_cache_tests {
         const S0_VIEW_SIZE: u64 = 4;
 
         let s0 = ShardUId::single_shard();
-        store_config.trie_cache.default_max_bytes = DEFAULT_SIZE;
-        store_config.trie_cache.per_shard_max_bytes.insert(s0, S0_SIZE);
-        store_config.view_trie_cache.default_max_bytes = DEFAULT_VIEW_SIZE;
-        store_config.view_trie_cache.per_shard_max_bytes.insert(s0, S0_VIEW_SIZE);
+        store_config.trie_cache.default_max_bytes = bytesize::ByteSize(DEFAULT_SIZE);
+        store_config.trie_cache.per_shard_max_bytes.insert(s0, bytesize::ByteSize(S0_SIZE));
+        store_config.view_trie_cache.default_max_bytes = bytesize::ByteSize(DEFAULT_VIEW_SIZE);
+        store_config
+            .view_trie_cache
+            .per_shard_max_bytes
+            .insert(s0, bytesize::ByteSize(S0_VIEW_SIZE));
```

Comment:
As `ByteSize` exposes its constructor, we could have the cast already be in the `const` expressions just above :)

Reply:
good point, done!